### DISCUSSION TITLE
注册表文件全部的细节的提问

### DISCUSSION THREAD

**Discussion by misividkoukou** [Meloong-Git/PCL#1850] [提问] [answered]
希望可以知道在同一电脑与不同电脑注册表的全部用途，试问不同电脑同一个人隐藏主题可以迁移吗

**Comment by WForst-Breeze** [comment, marked as answer]
PCL2 会在注册表中储存部分隐私设置，包括隐藏主题、账户信息、Java 路径、游戏文件夹路径等。其中前两者——即隐藏主题和账号登录信息——不可以在更换电脑的情况下迁移。具体请见内置帮助库 **启动器-备份启动器设置**

**Reply by Pigeon0v0** [reply]
PCL2 并不会存储微软账号和密码，而只会存储 OAuth 获取到的 refreshToken 等 Token 信息

**Reply by WForst-Breeze** [reply]
好的，明白啦🧐

**Comment by cancatchina** [comment]
各位大佬，如果我不想玩了，应当如何清除注册表呢

**Reply by lactobionicAcid** [reply]
其实没必要清除注册表，这个注册表并不会对电脑造成任何影响……

但如果真要清除的话，按下 <kbd>Windows</kbd> + <kbd>R</kbd> 键呼出运行，然后输入 `REG DELETE HKEY_CURRENT_USER\Software\PCL /f`。
请注意在清除注册表后，里面的所有信息会永久丢失，**下手前请慎重考虑。**
